distribution/packages/src/deb/debian/preinst

```diff
@@ -13,10 +13,14 @@ set -e
 
 echo "Running Wazuh Indexer Pre-Installation Script"
 
+# Reference to restore actual service status
+restart_service=/tmp/wazuh-indexer.restart
+
 # Stop existing service
 if command -v systemctl >/dev/null && systemctl is-active wazuh-indexer.service >/dev/null; then
     echo "Stop existing wazuh-indexer.service"
     systemctl --no-reload stop wazuh-indexer.service
+    touch $restart_service
```

Comment:
Consider using [systemd-tmpfiles](https://www.baeldung.com/linux/systemd-tmpfiles-configure-temporary-files)

Reply:
Analysis in the comment https://github.com/wazuh/wazuh-indexer/pull/492#issuecomment-2441861588